apps/web/src/pages/Applications/ApplicationQuestionsPage/ApplicationQuestionsPage.tsx

```diff
@@ -66,9 +76,166 @@ export const getPageInfo: GetApplicationPageInfo = ({
+      },
+    })
+      .then((res) => {
+        if (!res.error) {
+          toast.success(
+            intl.formatMessage({
+              defaultMessage:
+                "Successfully updated screening question responses!",
+              id: "kJUKrT",
+              description:
+                "Message displayed to users when saving screening question responses is successful.",
+            }),
+          );
+          navigate(paths.applicationReview(application.id));
+        }
+      })
+      .catch(() => {
+        toast.error(
+          intl.formatMessage({
+            defaultMessage: "Error: adding experience failed",
+            id: "moKAQP",
+            description:
+              "Message displayed to user after experience fails to be created.",
+          }),
+        );
+      });
+  };
+
+  return (
+    <>
+      <div
+        data-h2-display="p-tablet(flex)"
+        data-h2-align-items="p-tablet(flex-end)"
+        data-h2-justify-content="p-tablet(space-between)"
+        data-h2-margin-bottom="base(x1)"
+      >
+        <Heading data-h2-margin="base(0)" data-h2-line-height="base(1)">
+          {pageInfo.title}
+        </Heading>
+        <Link
+          mode="inline"
+          type="button"
+          color="secondary"
+          href={paths.applicationQuestionsIntro(application.id)}
+        >
+          {intl.formatMessage({
+            defaultMessage: "Review instructions",
+            id: "VcpIlx",
+            description:
+              "Link text to return to an introduction page on an application",
+          })}
+        </Link>
+      </div>
+      <p data-h2-margin="base(x1, 0)">
+        {intl.formatMessage({
+          defaultMessage:
+            'Don\'t forget to take a break if you need to! Using the "Save and quit for now" button, you can record your progress and return to questions you might be stuck on later.',
+          id: "tMnjyJ",
+          description:
+            "Notice that application can be saved and returned to at a later time.",
+        })}
+      </p>
+      <BasicForm
+        onSubmit={handleSubmit}
+        options={{
+          defaultValues: dataToFormValues(
+            screeningQuestions,
+            screeningQuestionResponses,
+          ),
+        }}
+      >
+        {screeningQuestions.length ? (
+          screeningQuestions.map((question, index) => (
+            <React.Fragment key={question.id}>
+              <Heading level="h3" size="h4">
+                {intl.formatMessage(
+                  {
+                    defaultMessage: "Question {number}",
+                    id: "/sBGov",
+                    description: "Heading for a specific screening question",
+                  },
+                  { number: index + 1 },
+                )}
+              </Heading>
+              <input type="hidden" name={`answers.${index}.id`} />
+              <input type="hidden" name={`answers.${index}.questionId`} />
+              <TextArea
```

Comment:
I'm not sure the question text should be the label.  It looks like it should be a paragraph above the input.
![image](https://user-images.githubusercontent.com/8978655/233648407-0db464c6-a261-45c2-b8f1-79c263c2deda.png)

Reply:
I left a response on the issue itself. I feel like we need to find a good balance between looking nice and allowing it to work well for all users 👍

Reply:
Based on feedback from @substrae this has been updated to use both the question and "answer to..." as labels for the text area in [update to use two labels for questions](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6311/commits/ce87567042afc2de9d1d5b2f7cb1cd362e0cf063)

It should now read to ATS as:

"Some question we are asking? Answer to questions 1. Required, editable multiline, blank."